pkg/e2e/operators/managedvelero.go

```diff
@@ -56,7 +56,7 @@ var _ = ginkgo.Describe(veleroOperatorTestName, ginkgo.Ordered, label.Operators,
 	checkRoleBindings(h,
 		"kube-system",
 		[]string{"managed-velero-operator-cluster-config-v1-reader"})
-	checkVeleroBackups(h)
+	checkVeleroBackups(h, operatorNamespace)
```

Comment:
:laughing: this function has been using the wrong namespace, it was pointing to a global variable in the operator package